ensemble-view/src/main/java/hec/ensembleview/ComponentsPanel.java

```diff
@@ -42,11 +42,16 @@ private void addComponentsToTimeSeriesViewPanel() {
                 case CUMULATIVE:
                     timeSeriesViewPanel.add((RadioButtonStat)stat);
                     buttonGroup.add(((RadioButtonStat)stat).getRadioButton());
+                case PLOTTINGPOSITION:
```

Comment:
The logic in the "PLOTTINGPOSITION" case appears to repeat what is in the "CUMULATIVE" case. Should these cases be merged?

Reply:
Merged the cases together